node/src/neighborhood/gossip_acceptor.rs

```diff
@@ -3264,10 +3609,10 @@ mod tests {
             dest_db.add_arbitrary_half_neighbor(dest_node.public_key(), failed_node_key);
```

Comment:
I'd also appreciate if we looked finally reliably into the comments left over here. There are two. One about a revealed Ip and another about half neighbors and degree calculations. 

Please, understand, I don't want to let this uttered uncertainty about the Gossip effects live on too long. This will serve indirectly as documentation of the processes and we should make it as accurate as possible. Probably Dan again needed for his assistance here.